### ISSUE TITLE
Add fstab cheat sheet

### ISSUE THREAD

**Issue by chubin** [chubin/cheat.sheets#156]
@carlogarro wrote:

```
It could be nice to have a guide for automounting devices. And for that, [fstab](https://en.wikipedia.org/wiki/Fstab) is necessary.
Just giving some ideas.

Amazing work, by the way!
```

(from github.com/chubin/cheat.sh#281)

**Comment by terminalforlife** [comment]
I'd put that in a subdirectory, since it would otherwise suggest and perhaps conflict with a program, present or future, called `fstab`.

**Comment by chubin** [comment]
I think we already have some conflicts like that, but it is not so critical. I think it is fine to have in the main namespace. Later we could move the config files to namespaces, according to their locations, e.g. `cheat.sh//etc/fstab` or `cheat.sh/etc/fstab`.
I suppose almost all of them will be in `etc`. But for the moment simply `fstab` is fine

**Comment by terminalforlife** [comment]
Looking at this again... isn't `man fstab` enough? It seems to do such a good job at explaining the fields. It even explains the use of `\040` for spaces, which I honestly wasn't expecting to see.

**Comment by chubin** [comment]
I agree that sometimes man is enough, but sometimes it is wort to it have a cheat sheet.
For me, the best indication whether we need a cheat sheet or not, is how often users are trying to access it.
We have this data, but it is not yet well-structured and analyzed, but I hope we will do it, and the results will be available at some cheat.sh page
